tests/RobotTest.php

```diff
@@ -189,4 +189,25 @@ public function testHandlesEmptyUserAgentLine()
         $this->assertFalse($g->isAllowed('foo', '/something'));
         $this->assertTrue($g->isAllowed('Googlebot', '/something'));
     }
+
+    /**
+     * Sometimes we might get rubbish with a :: in it, which will confuse the parser...
+     */
+    public function testHtmlAndCSS() {
+        $html = <<<EOF
```

Comment:
The other tests mainly use external files in `files/` for their input data, would you mind moving this for consistency?
Also what was the previous behaviour with double colons? Was that when the notice was thrown?

Sorry about all the questions, its been a while since I last wrote PHP 😄

Reply:
The previous behaviour was that it'd return "" and emit an undefined offset notice. 

It now just doesn't emit the notice.